sources/packages/api/src-sql/EducationPrograms/Alter-education-programs.sql

```diff
@@ -0,0 +1,159 @@
+--Drop the NOT NULL constraint on program_description
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    program_description DROP NOT NULL;
+
+--Changing the data type of has_minimun_age as it is one of entrance requirement now.
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    has_minimun_age TYPE BOOLEAN USING CASE
```

Comment:
Instead of doing an alter can we recreate the column and fix the typo on _has_minimun_age_?

Reply:
Column recreate scripts added.

Reply:
Thanks for taking care of it 😉